mscore/plugin/mscorePlugins.cpp

```diff
@@ -362,36 +362,85 @@ bool MuseScore::loadPlugin(const QString& filename)
       }
 
 //---------------------------------------------------------
-//   pluginTriggered
+//   pluginPathFromIdx
 //---------------------------------------------------------
 
-void MuseScore::pluginTriggered(int idx)
+QString MuseScore::pluginPathFromIdx(int idx)
       {
       if (plugins.size() > idx)
-            pluginTriggered(plugins[idx]);
+            return plugins[idx];
+      else
+            return nullptr;
```

Comment:
I meant to return something that shows the plugin isn't loaded, what's usualy used?

Reply:
> You added some functionality in third party libraries. I know that we tend to refrain from doing that as much as possible, however, if they are cloned into the repo, it's probably because we still need to modify them sometimes. The others will know more about it.
> Also, you where is the documentation for using what you just did with osc? How could i find what to do if I wanted to call a plugin?

I will add a proper documentation but i wanted first to know if theses changes are correct and interesting, there is an explanation on the MuseScore.org Issue.

Reply:
It doesn't really return a string (`char[]` or `char*`) but a `QString`. Those can come ans an empty or as none at all. Probably better to `return QString();`?